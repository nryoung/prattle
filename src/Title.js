import React from 'react';
import styled from 'styled-components';

const StyledTitle = styled.div`
  font-size: 72px;
  color: red;
`;

const Title = () => <StyledTitle>Title</StyledTitle>;

export default Title;
