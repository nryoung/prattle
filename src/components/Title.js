import React from 'react';
import styled from 'styled-components';
import * as colors from '../styles/colors';

const StyledTitle = styled.div`
  position: fixed;
  top: 0;
  width: 100vw;
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: ${colors.powersuit};
  height: 56px;
`;

const TitleLink = styled.a`
  font-size: 32px;
  color: ${colors.pureWhite};
  text-decoration: none;

  &:hover {
    text-decoration: underline;
  }
`;

const Title = () => (
  <StyledTitle>
    <TitleLink href="/">Prattle</TitleLink>
  </StyledTitle>
);

export default Title;
