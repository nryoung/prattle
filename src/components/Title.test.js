import React from 'react';
import renderer from 'react-test-renderer';
import Title from './Title';

it('it renders', () => {
  const component = renderer.create(<Title />);
  expect(component.toJSON()).toMatchSnapshot();
});
