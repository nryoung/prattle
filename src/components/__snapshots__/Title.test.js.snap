// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="sc-bdVaJa bkgjlC"
>
  <a
    className="sc-bwzfXH fhnlUk"
    href="/"
  >
    Prattle
  </a>
</div>
`;
